usmqe_tests/api/ceph/test_ceph_cluster.py

```diff
@@ -112,11 +113,16 @@ def test_cluster_import_valid(valid_session_credentials):
         section="parameters")
 
     LOGGER.debug("integration_id: %s" % integration_id)
-    pytest.check(
-        [x for x in api.get_cluster_list()
-         if x["integration_id"] == integration_id],
-        "Job list integration_id '{}' should be "
-        "present in cluster list.".format(integration_id),
-        issue="https://github.com/Tendrl/api/issues/154")
+    try:
+        pytest.check(
+            [x for x in api.get_cluster_list() if x.get("integration_id", "") == integration_id],
```

Comment:
Yes, integration_id is unique ID for cluster in Tendrl. Ceph cluster can be identified also according its name.